docs/getting_oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,170 @@
+# Getting the OSS rules of play ratings
+
+The page describes how a rules of play rating may be calculated for an open source project.
+
+## Configuration
+
+Several of the rules need to configured as the detailed guidelines differ from project to project, for example which licenses may be allowed in your organization or which sections you want to enforce in the README file. Thus, in order to get proper OSS rules of play ratings, you need to provide some details. These details are usually passed to Fosstars via YAML configuration files for the respective data providers. Below you find the configurable data providers, information how to name the configuration files and the possible options in the respective configuration file. The section about the command-line tool further below explains how to pass these configuration files to Fosstars. The configuration that SAP uses for their OSS rules of play check run can be accessed in the [fosstars-oss-rules-of-play-sap-report](https://github.com/SAP/fosstars-rating-core/tree/fosstars-oss-rules-of-play-sap-report) branch.
+
+### Contributing Guidelines
+This data provider with the ID `ContributingGuidelineInfo` allows you to specify regular expression patterns with content that you expect to be in the contributing guidelines or that must not be in the contributing guidelines of the analyzed project. The option is called `requiredContentPatterns` and expects a list of one or multiple entries. If you enter multiple entries, the contribution guidelines must match all patterns for the rule to pass.
+
+### Licensing
+This data provider with the ID `LicenseInfo` allows you to specify which license(s) should be allowed in the analyzed project. This option is called `allowedLicenses` and expects a list of one or more SPDX license IDs. In addition you can check for terms that must not in the license (e.g. to avoid modifications) using regular expression patterns. The corresponding option is called `disallowedLicensePatterns`, accepts a list of patterns, but can also be left empty. Finally, you are also able to specify exceptions, i.e. which repositories shall not be checked with this data provider, for example because certain repositories are known to use other licenses, but have gone through an exception process. The option is called `repositoryExceptions` and can accept a list of full repository URLs, e.g. `https://github.com/SAP/fosstars-rating-core`, but can also be left empty if there are no exceptions.
```

Comment:
The data provider `LicenseInfo`

Reply:
There are few redundant lines like 
- `but can also be left empty`.
- Could we split this into steps or points. When we use Finally, it usually means there is few steps above or few options which should be followed.

Reply:
Yeah, also thought about bullet points, but for now I kept it like that as it suits a little better to the style of the rest. The "can also be left empty" is there to emphasize that it's an optional entry.

Reply:
```suggestion
The data provider `LicenseInfo` allows you to specify which license(s) should be allowed in the analyzed project.
This option is called `allowedLicenses` and expects a list of one or more SPDX license IDs.
In addition you can check for terms that must not in the license (e.g. to avoid modifications) using regular expression patterns.
The corresponding option is called `disallowedLicensePatterns`, and it accepts a list of patterns, but can also be left empty.
Finally, you are also able to specify exceptions, i.e. which repositories shall not be checked with this data provider, for example, because certain repositories are known to use other licenses, but have gone through an exception process.
The option is called `repositoryExceptions` and can accept a list of full repository URLs, e.g. `https://github.com/SAP/fosstars-rating-core`, but can also be left empty if there are no exceptions.
```